### DISCUSSION TITLE
Index Error while running the Brownie_Fund_Me project

### DISCUSSION THREAD

**Discussion by InsideTrifle** [smartcontractkit/full-blockchain-solidity-course-py#1445] [Q&A] [answered]
```
brownie run scripts/fund_and_withdraw.py --network ganache-local

Brownie v1.18.1 - Python development framework for Ethereum

BrownieFundMeProject is the active project.

Running 'scripts/fund_and_withdraw.py::main'...
  File "brownie/_cli/run.py", line 51, in main
    return_value, frame = run(
  File "brownie/project/scripts.py", line 103, in run
    return_value = f_locals[method_name](*args, **kwargs)
  File "./scripts/fund_and_withdraw.py", line 22, in main
    fund()
  File "./scripts/fund_and_withdraw.py", line 6, in fund
    fund_me = FundMe[-1]
  File "brownie/network/contract.py", line 175, in __getitem__
    return self._contracts[i]
IndexError: list index out of range`
```

I dont get why this is not working from the Brownie Fund Me project
everything seems alright to me and to double check I even cloned Patricks repo and ran it. Same problem on that one as well.

Any help would be appreciated

**Comment by cromewar** [comment, marked as answer]
Hello @InsideTrifle 
using the index [-1] just works for networks with persistence, meaning a testnet or a ganache with pseudo-persistence. If you run this just on development it won't work, as the [-1] looks into the build folder to find the address of the deployed contracts, and using development them will not ever be created.

**Reply by InsideTrifle** [reply]
Thanks I got it

**Comment by coodered** [comment]
Thanks so much dear

**Comment by coodered** [comment]
So I'm a beginner I need to develop fastly my skills as Blockchain and
smart contracts development, and sucre assets. Can you help me along this
challenge please?

**Reply by cromewar** [reply]
Could you be more specific on your questions please?
